scripts/generate/readme-repo.ts

```diff
@@ -18,14 +18,14 @@ export const generateMainReadme = async (
   if (!portsData.categories) throw ("Ports data is missing categories");
 
   const categorized = Object.entries(userstyles)
-    .reduce((acc, [slug, { category, ...port }]) => {
+    .reduce((acc, [slug, { categories, ...port }]) => {
       // initialize category array if it doesn't exist
-      acc[category] ??= [];
+      acc[categories[0]] ??= [];
```

Comment:
do you think it is worth noting why we are using `categories[0]`

Reply:
I suppose it can't hurt, I left it out because the schema explains that the [first category will be taken](https://github.com/catppuccin/userstyles/pull/559/files#diff-38bad6586853d69c74fa1368f7fda7f03234fcc4d6574f6bc4fb8d9f8a187779R43) but it definitely doesn't make sense by its own?